### PR TITLE
add xarray-selafin

### PR DESCRIPTION
**Pull request by tomsail**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tomsail** [comment]
pinging @pmav99

**Comment by pmav99** [comment]
I confirm that I am willing to maintain this package

**Comment by pmav99** [comment]
@conda-forge-admin, please lint

**Comment by pmav99** [comment]
@conda-forge/help-python, ready for review

File: recipes/xarray-selafin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "xarray_selafin" %}
+{% set version = "0.1.8" %}
+{% set sha256 = "7e1e952cd4f3c4dcfc5063508e528bec0d7bea0c208e005e3952d29a23e264a2" %}
```
Comment (ocefpaf) [inline recipes/xarray-selafin/meta.yaml:3]:
```suggestion
```

File: recipes/xarray-selafin/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/xarray-selafin/meta.yaml:11]: Don't template single use variables. That obfuscates the recipe and doesn't really help. users.

```suggestion
  sha256: 7e1e952cd4f3c4dcfc5063508e528bec0d7bea0c208e005e3952d29a23e264a2
```

File: recipes/xarray-selafin/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/xarray-selafin/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Pull request merged**